src/components/ui/Auditor/AuditorImport.svelte

```diff
@@ -5,6 +5,7 @@
     label="{TRANSLATED.BUTTON}"
     labelsub="({TRANSLATED.BUTTON_HINT})"
     on:change="{handleChange}"
+    on:click={onInputClick}
```

Comment:
Maybe call that function `handleClick` to make it more similar to `handleChange`?